streams/dlna_meta.py

```diff
@@ -1,68 +1,210 @@
 #!/usr/bin/python3
```

Comment:
This line should be removed so we can launch this with the same version of python that amplipi is running